mne_bids/write.py

```diff
@@ -1922,28 +1929,100 @@ def mark_bad_channels(ch_names, descriptions=None, *, bids_path,
     >>> root = Path('./mne_bids/tests/data/tiny_bids').absolute()
     >>> bids_path = BIDSPath(subject='01', task='rest', session='eeg',
     ...                      datatype='eeg', root=root)
-    >>> mark_bad_channels('C4', bids_path=bids_path, verbose=False)
+    >>> mark_channels('C4', bids_path=bids_path, verbose=False)
 
     Mark multiple channels as bad, and add a description as to why.
 
     >>> bads = ['C3', 'PO10']
     >>> descriptions = ['very noisy', 'continuously flat']
-    >>> mark_bad_channels(bads, descriptions, bids_path=bids_path,
+    >>> mark_channels(bads, descriptions, bids_path=bids_path,
     ...                   verbose=False)
 
     Mark two channels as bad, and mark all others as good by setting
     ``overwrite=True``.
 
     >>> bads = ['C3', 'C4']
-    >>> mark_bad_channels(bads, bids_path=bids_path,
+    >>> mark_channels(bads, bids_path=bids_path,
     ...                   overwrite=True, verbose=False)
 
     Mark all channels as good by passing an empty list of bad channels, and
     setting ``overwrite=True``.
 
-    >>> mark_bad_channels([], bids_path=bids_path, overwrite=True,
+    >>> mark_channels([], bids_path=bids_path, overwrite=True,
     ...                   verbose=False)
 
+    """
+    mark_channels(ch_names, descriptions=descriptions, bids_path=bids_path,
+                  status='bad', overwrite=overwrite, verbose=verbose)
+
+
+@verbose
+def mark_channels(ch_names, descriptions=None, *, bids_path,
+                  status='bad', overwrite=False, verbose=None):
+    """Update status and description of channels in an existing BIDS dataset.
+
+    Parameters
+    ----------
+    ch_names : str | list of str
+        The names of the channel(s) to mark as bad. Pass an empty list in
+        combination with ``overwrite=True`` to mark all channels as good.
```

Comment:
is this behavior still true?

Reply:
Yes I believe so, since unit tests for this behavior still work. Do you think the behavior should change?

Reply:
I guess it's weird because this means it ignores the argument `statuses` when `ch_names=[]`